src/ini.cr

```diff
@@ -23,21 +23,20 @@ class INI
   # Generates an INI-style configuration from a given hash.
   #
   # ```
-  # INI.build({"foo" => {"a" => "1"}}, " ") # => "[foo]\na = 1\n\n"
+  # INI.build({"foo" => {"a" => "1"}}, true) # => "[foo]\na = 1\n\n"
   # ```
-  def self.build(ini, space : String = "") : String
-    String.build do |str|
-      build str, ini, space
-    end
+  def self.build(ini, space : Bool = false) : String
+    String.build { |str| build str, ini, space }
   end
 
   # Appends INI data to the given IO.
   #
-  def self.build(io : IO, ini, space : String = "")
+  def self.build(io : IO, ini, space : Bool = false)
+    s = space ? " " : ""
     ini.each do |section, contents|
       io << '[' << section << "]\n"
       contents.each do |key, value|
-        io << key << space << '=' << space << value << '\n'
+        io << key << s << '=' << s << value << '\n'
```

Comment:
I'd rather
```cr
io << key
io << ' ' if space
io << '='
io << ' ' if space
io << value << '\n'
```

Reply:
I've done it with `io << key << (space ? " = " :  '=') << value << '\n'`, less verbose. Is it fine?